snippets/numpy/core/numeric.py

```diff
@@ -0,0 +1,59 @@
+from ulab import numpy as np
+from .multiarray import (asarray)
+
+def zeros_like(a, dtype=None, order='K', subok=True, shape=None):
```

Comment:
We have `zeros`. Is this simply an alias, so that you can deal with the keyword arguments?

Reply:
The difference is the shape.  https://numpy.org/doc/stable/reference/generated/numpy.zeros_like.html

Reply:
We could implement it in C, but it wouldn't actually result in speed gains, so I would leave this here.